src/patternfly/components/Alert/examples/alert-danger-example.hbs

```diff
@@ -1,11 +1,17 @@
 {{#> alert alert--modifier="pf-m-danger" alert--attribute='aria-label="Danger Notification"'}}
   {{#> alert-icon alert-icon--danger="true"}}
   {{/alert-icon}}
-    {{#> alert-body alert-HasDescription="true"}}
-      {{#> alert-title}}
+  {{#> alert-body}}
+    {{#> alert-title}}
       {{#> accessibility accessibility--type="sr-only"}}Danger: {{/accessibility}}
         Danger notification title
-      {{/alert-title}}
+    {{/alert-title}}
+    {{#> alert-description}}
+			Danger notification description.
+		{{/alert-description}}
+    {{#> alert-link alert-link--modifier="pf-c-button pf-m-link"}}
```

Comment:
FYI, this creates `.pf-c-alert-link.pf-c-button`. Assuming you want to create a `.pf-c-alert__link` element for the Alert component, this element shouldn't also be another component. You would want to apply `.pf-c-alert__link` to a `button` element and style that as needed, or use a `.pf-c-button` inside of `.pf-c-alert__link`

Reply:
My suggestion would be that if the link is a link that flows with the sentence, especially if it's something that takes you off page to some sort of resource like documentation, leave it as a regular link in the description text. However, if it's the primary action of the alert, especially if it performs an action, that should be a button in `.pf-c-alert__action`

Reply:
great, thanks for the clarification!